src/main/helpers.ts

```diff
@@ -0,0 +1,116 @@
+import { dungeonsByMapId, instanceDifficulty, InstanceDifficultyType, instanceEncountersById, instanceNamesByZoneId, raidInstances, VideoCategory, zones } from "./constants";
+import { Metadata } from "./logutils";
+import { RaidInstanceType } from "./types";
+
```

Comment:
Can we get a comment in here to be explicit about no filesystem stuff in this file.